common/tier4_autoware_utils/include/tier4_autoware_utils/ros/parameter.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TIER4_AUTOWARE_UTILS__ROS__PARAMETER_HPP_
+#define TIER4_AUTOWARE_UTILS__ROS__PARAMETER_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+namespace tier4_autoware_utils
+{
+template <class T>
+T getOrDeclareParameter(rclcpp::Node & node, const std::string & name)
+{
+  if (node.has_parameter(name)) {
+    return node.get_parameter(name).get_value<T>();
+  }
+
+  try {
+    return node.declare_parameter<T>(name);
+  } catch (const rclcpp::ParameterTypeException & ex) {
+    RCLCPP_ERROR(
+      node.get_logger(), "Failed to get parameter `%s`, please set it when you launch the node.",
+      name.c_str());
+    throw(ex);
```

Comment:
This error or warning is shown by default. Any reason to add this?

Reply:
Good. I just copied it from another pkg, but it can be removed.

Reply:
@takayuki5168 removed. Thank you!